go/chat/utils/utils.go

```diff
@@ -796,30 +796,37 @@ func systemMessageSnippet(msg chat1.MessageSystem) string {
 }
 
 func GetMsgSnippet(msg chat1.MessageUnboxed, conv chat1.ConversationLocal, currentUsername string) string {
-	if !msg.IsValidFull() {
-		if msg.IsValid() {
-			mvalid := msg.Valid()
-			if mvalid.IsEphemeral() && mvalid.IsEphemeralExpired(time.Now()) {
-				return "[exploded message 💥]"
-			}
-		}
+	if !msg.IsValid() {
 		return ""
 	}
-	var prefix string
-	switch conv.GetMembersType() {
-	case chat1.ConversationMembersType_TEAM:
-		sender := msg.Valid().SenderUsername
+
+	mvalid := msg.Valid()
+	var senderPrefix string
+	if len(conv.Names()) > 2 {
```

Comment:
Think team should always show it. Just change the switch back to yielding like a bool of whether or not to append the sender.

Reply:
updated!